src/main/java/com/kammatysiak/medicalclinic/service/PatientService.java

```diff
@@ -0,0 +1,82 @@
+package com.kammatysiak.medicalclinic.service;
+
+import com.kammatysiak.medicalclinic.exceptions.PatientDoesNotExistException;
+import com.kammatysiak.medicalclinic.exceptions.PatientExistsException;
+import com.kammatysiak.medicalclinic.exceptions.PatientNullFieldException;
+import com.kammatysiak.medicalclinic.mapper.PatientMapper;
+import com.kammatysiak.medicalclinic.model.dto.PasswordClassDTO;
+import com.kammatysiak.medicalclinic.model.dto.PatientCreateDTO;
+import com.kammatysiak.medicalclinic.model.dto.PatientDTO;
+import com.kammatysiak.medicalclinic.model.entity.Patient;
+import com.kammatysiak.medicalclinic.repository.PatientRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.kammatysiak.medicalclinic.validator.PatientValidator.*;
+
+@RequiredArgsConstructor
+@Service
+public class PatientService {
+
+    private final PatientRepository patientRepository;
+
+    private final PatientMapper patientMapper;
+
+    public List<PatientDTO> getPatients() {
+        List<Patient> patientList = patientRepository.findAll();
+        List<PatientDTO> patientDTOList = new ArrayList<>();
```

Comment:
nie potrzebujesz tego. 
Zrób tak abyś mial
return i operacje na streamie, która ci to robi.

Reply:
return patientList.stream()
.
. 
.